test/e2e/specs/specs-playwright/wp-widgets__spec.ts

```diff
@@ -0,0 +1,50 @@
+import { DataHelper, LoginFlow, SidebarComponent, setupHooks } from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+const gutenbergUser =
+	process.env.GUTENBERG_EDGE === 'true' ? 'gutenbergSimpleSiteEdgeUser' : 'gutenbergSimpleSiteUser';
+
+describe( DataHelper.createSuiteTitle( 'Widgets' ), function () {
+	let sidebarComponent: SidebarComponent;
+	let page: Page;
+
+	setupHooks( ( args ) => {
+		page = args.page;
+	} );
+
+	it( 'Log in', async function () {
+		const loginFlow = new LoginFlow( page, gutenbergUser );
+		await loginFlow.logIn();
+	} );
+
+	// @todo: Refactor/Abstract these steps into a WidgetsEditor component
+	it( 'Navigate to the Block Widgets Editor', async function () {
+		sidebarComponent = new SidebarComponent( page );
+		await sidebarComponent.gotoMenu( {
+			item: 'Appearance',
+			subitem: 'Widgets',
+			skipWaitForContent: true,
+		} );
+	} );
+
+	it( 'Dismiss the Welcome Gudie Notice', async function () {
```

Comment:
non-blocker: Since this step may be optional, it may be a good idea to name this step something like 

`Dismiss the Welcome Guide if present`.

Also, typo in the word `Guide`.

Reply:
Yeah, I had name it with the `if present` suffix, though I found out it is always shown.

Reply:
Not sure if it's a bug though, it appears it should be a one-time thing, so I'd still keep the condition that check for the button meanwhile. I'll ask around to find out what's the correct behavior here.

Reply:
Sounds good! Not a blocker by any means.